### ISSUE TITLE
Old key not deleted when run via cron/brew services

### ISSUE THREAD

**Issue by acceptableEngineering** [rhyeal/aws-rotate-iam-keys#50]
The latest stable version of the AWS CLI SDK requires user interaction when `aws iam delete-access-key` is run on [line 168](https://github.com/rhyeal/aws-rotate-iam-keys/blob/master/src/bin/aws-rotate-iam-keys#L168)

Therefore, when your (awesome!) script is run via `crontab` or `brew services` and there is no user interaction as a result, the old key is never deleted. This behavior can be seen by using the AWS CLI SDK directly:
```
$  aws --version
aws-cli/2.0.58 Python/3.9.0 Darwin/19.6.0 source/x86_64

$  aws iam delete-access-key --access-key-id ACCESS_KEY_ID_HERE
# Prompts for user interaction here, requiring :q to be input
```

**Comment by acceptableEngineering** [comment]
Because I believe that ultimately this bug is the fault of the AWS CLI, I've opened a bug report over there: https://github.com/aws/aws-cli/issues/5668

I'm hoping that my bug report here on `aws-rotate-iam-keys` might prompt logic that catches and handles their error

**Comment by mmrwoods** [comment]
@acceptableEngineering Are you sure this is happening when the script is run via cron or homebrew services?

A pager is typically only used when the standard output stream is a terminal.

I've pushed a branch that should fix it if there is a bug with the AWS CLI that results in the pager being used even when stdout is not a tty.

See https://github.com/rhyeal/aws-rotate-iam-keys/pull/52

**Comment by mmrwoods** [comment]
Fixed, see #52
